opm/autodiff/FlowMainEbos.hpp

```diff
@@ -707,8 +700,17 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        Problem& ebosProblem()
```

Comment:
I think only the const version is required at the moment, but possibly some non-const method of the EclProblem needs to be called in the future. (it does not hurt to have both versions, IMO.)